core/src/main/java/apoc/trigger/TriggerHandler.java

```diff
@@ -287,6 +288,11 @@ public void afterRollback(TransactionData txData, Void state, GraphDatabaseServi
         );
     }
 
+    private <E> List<E> toList(Iterable<E> iterable) {
```

Comment:
If there is a preferred method for this in one of the included libraries we can remove this method and use that